packages/trpc/server/routers/viewer/teams/legacyListMembers.schema.ts

```diff
@@ -7,9 +7,7 @@ export const ZListMembersInputSchema = z.object({
   cursor: z.number().nullish(),
 });
 
-export const ZLegacyListMembersInputSchema = ZListMembersInputSchema.extend({
-  includeEmail: z.boolean().optional(),
-});
+export const ZLegacyListMembersInputSchema = ZListMembersInputSchema.extend({});
```

Comment:
can't this be just 

export const ZLegacyListMembersInputSchema = ZListMembersInputSchema

Reply:
or just use ZListMembersInputSchema

Reply:
@Udit-takkar thanks! I've actually deleted this in #19445 just now (this is a stacked PR on top of it)